config/initializers/devise.rb

```diff
@@ -183,7 +183,7 @@
   # Email regex used to validate email formats. It simply asserts that
   # one (and only one) @ exists in the given string. This is mainly
   # to give user feedback and not to assert the e-mail validity.
-  config.email_regexp = /\A[^@\s]+@[^@\s]+\z/
+  config.email_regexp = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
```

Comment:
Should we add a test for it? 🤔 

Another possibility is to add a higher level test in `spec/forms/registration_form_spec.rb` with something like `given an invalid email`/`it has the error message` ;-)